### PR TITLE
used is-git-url lib to check the github url and send 500 error response

### PR DESCRIPTION
**Pull request by soumyanayak95**

### CODE REVIEW

**Comment by soumyanayak95** [comment]
post body is x-www-form-urlencoded format: url: "github-url"

File: api/src/Routes/testAPI.js [hunk 1]
```diff
@@ -22,51 +17,25 @@ router.get("/test", function (req, res, next) {
...
-  exec("bash.sh " + `'${u}'`, (err, stdout, stderr) => {
-    if (err) {
-      console.error(err);
-      return;
-    }
-    console.log(stdout);
-  });
-
-  //res.send("Bash script executed here");
+  const url = req.body.url;
```
Comment (Bhabaranjan19966) [inline api/src/Routes/testAPI.js:22]: Please, change the route name as mentioned in the issue. 
Use try  and catch block to handle errors.

File: api/src/Routes/testAPI.js [hunk 2]
```diff
@@ -22,51 +17,25 @@ router.get("/test", function (req, res, next) {
...
+  console.log(url);
+  if ( !isGitUrl(url) ){
+    console.log("Invalid Git url!!!")
+    res.status(500).send("Invalid Git url");
+  }
+  else{
+    const ls = spawn("git", ['clone', url]);
+    ls.stdout.on("data", (data) => console.log("data", data.toString('utf-8')));
+    ls.stderr.on('data', (data) => {
+        console.log(`stderr: ${data}`);
```
Comment (Bhabaranjan19966) [inline api/src/Routes/testAPI.js:32]: Use console.error for logging errors.

**Comment by Bhabaranjan19966** [comment]
@soumyanayak95  Please remove this health check, as we are going to add this at the `docker service level` going forward, not at the server level.
```
router.get("/", function (req, res, next) {
  res.send("API is working properly in the GET method");
});
```

**Comment by soumyanayak95** [comment]
try-catch added

**Pull request merged**